src/components/templates/SinopiaResourceTemplates.jsx

```diff
@@ -87,6 +108,7 @@ class SinopiaResourceTemplates extends Component {
     return (
       <div>
         { createResourceMessage }
+        { errorMessage }
```

Comment:
I assume you've verified locally that this doesn't double render the error?  For GroupChoiceModal, it did (b/c it reverted back to the Editor display)